alembic/versions/20250407_f36442df213d_add_data_source_name_to_playtime_.py

```diff
@@ -0,0 +1,64 @@
+"""Add data_source_name to playtime entries and summaries
+
+Revision ID: f36442df213d
+Revises: df27b4867e56
+Create Date: 2025-04-07 21:00:49.422754+00:00
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from palace.manager.sqlalchemy.model.time_tracking import PlaytimeEntry, PlaytimeSummary
```

Comment:
Alembic models aren't supposed to import sqlalchemy models, since these can change over time, but the migration should always stay the same.